pkgs/development/libraries/opencascade-occt_7-4/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, fetchpatch, cmake, ninja, tcl, tk,
+  libGL, libGLU, libXext, libXmu, libXi, darwin }:
+
+stdenv.mkDerivation rec {
+  pname = "opencascade-occt_7.4";
+  version = "7.4.0p1";
+  commit = "V${builtins.replaceStrings ["."] ["_"] version}";
+
+  src = fetchurl {
+    name = "occt-${commit}.tar.gz";
+    url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${commit};sf=tgz";
```

Comment:
```suggestion
  commit = "";

  src = fetchurl {
    url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=V${builtins.replaceStrings ["."] ["_"] version};sf=tgz";
```

Reply:
My idea was to keep it as close as possible to the current opencascade-occt package. Perhaps it should be changed there first?